data-structures-queues/get-front.js

```diff
@@ -0,0 +1,5 @@
+export default function getFront(queue) {
+  if (queue.peek() !== undefined) {
+    return queue.peek();
+  }
```

Comment:
You are relying on JS returning undefined if there is no return statement. This isn't usually safe or good code.

You could just `return queue.peek();`.